.travis.yml

```diff
@@ -57,6 +66,17 @@ after_success:
             docker push welder/web-with-coverage
             docker push welder/web-e2e-tests
         fi
+  - |
+        if [ "$COMMAND" == "cockpit-test" ]; then
```

Comment:
Move above and make the if statement something like ` if [ "$COMMAND" == "end-to-end-test"  ] || [ "$COMMAND" == "cockpit-test"] ; then`